.ci/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -24,6 +24,11 @@ call %INSTALLER_DIR%\install_sccache.bat
 if errorlevel 1 goto fail
 if not errorlevel 0 goto fail
 
+if "%USE_XPU%"=="1" (
+  :: Install oneAPI bundle
+  call %INSTALLER_DIR%\install_xpu.bat
+)
+
```

Comment:
Unfortunately, the current change can't build xpu wheel actually, refer https://github.com/pytorch/pytorch/actions/runs/10497376953/job/29080090269?pr=133151#step:11:623. Please follow the official bkc to activate bundle env

Reply:
Thanks for pointing out. I have added env var set up in the script as well.